app/models/session_activation.rb

```diff
@@ -28,7 +28,7 @@ class SessionActivation < ApplicationRecord
 
   before_create :assign_access_token
 
-  DEFAULT_SCOPES = %w(read write follow).freeze
+  DEFAULT_SCOPES = Doorkeeper::Application.webapp.scopes.without('push').freeze
```

Comment:
This change feels a bit weird, but I think it's the best way to keep the scopes consistent between `Doorkeeper::Application.webapp` and the access tokens for the webapp sessions.